### DISCUSSION TITLE
when 1.17 will release EssentialsX

### DISCUSSION THREAD

**Discussion by CRAFTINGHARD1234** [EssentialsX/Essentials#4357] [Q&A] [answered]
when 1.17 will release EssentialsX

**Comment by pop4959** [comment, marked as answer]
It's already out. Get the latest builds from the [download page](https://essentialsx.net/downloads.html).

**Reply by CRAFTINGHARD1234** [reply]
which one stable version?

**Reply by triagonal** [reply]
There is no stable release for 1.17 yet, only dev builds. The current dev builds are fairly stable though, so you should be fine to use them until the 2.19.0 update is released.

**Reply by CRAFTINGHARD1234** [reply]
> There is no stable release for 1.17 yet, only dev builds. The current dev builds are fairly stable though, so you should be fine to use them until the 2.19.0 update is released.

ok when it will release

**Reply by mdcfe** [reply]
> ok when it will release

There is no ETA. We roughly expect it to be soon, but don't have any specific date planned yet.
